src/lxc/network.c

```diff
@@ -470,9 +473,9 @@ static int setup_veth_ovs_bridge_vlan(char *veth1, struct lxc_netdev *netdev)
 	int taggedLength = lxc_list_len(&netdev->priv.veth_attr.vlan_tagged_ids);
 	struct ovs_veth_vlan_args args;
 	args.nic = veth1;
-	args.vlan_mode = "";
-	args.vlan_id = -1;
-	args.trunks = "";
+	args.vlan_mode = NULL;
```

Comment:
@brauner do you think coverity will also complain that these char pointers are not being freed (`args.vlan_mode` and `args.trunks`)?

Reply:
Yeah, you should free this memory if it is separate from `trunks`.